Assets/Scripts/Runtime/ItemPickCounter.cs

```diff
@@ -0,0 +1,18 @@
+﻿
+namespace Assets.Scripts.Runtime
+{
+    [System.Serializable]
+    public class ItemPickCounter
+    {
+        public BonusItemType _id;
+        public int _amount;
+
+        public string GetID { get { return _id.ToString(); } }
```

Comment:
ta funkcja powinna się nazywać `GetHudStringId()`
bo teraz jest mało czytelne co to za ID

konwersja `_id.ToString()`z enuma łatwo popsuć renamem enuma,
lepiej z palca wpisać switcha na stringi na ID HUDa zamiast polegać na nazwie enuma
zwłaszcza jak się potem coś doda i nie widać sugestii że na hudzie też trzeba dodać